### DISCUSSION TITLE
[Feature Request] Sort by Filename

### DISCUSSION THREAD

**Discussion by AhtanHigh** [PaulWoitaschek/Voice#1332] [Ideas]
Please include the following details in your bug report:

App-Version: 4.3.2
Phone Name: Wiki View Prime
Android Version: 7

Please can you add the possibility to Sort the Audiobook-List by Filename? 
In many cases Person have many audiobooks as Single-files on their Phone. And These audiobook are from a Series. To display this serie in correct order, IT is necessary to Sort the list by Filename, and not by title from id3tag. 

When i have Files like:

Audiobook ABC (1) - The Ruin.mp3
Audiobook ABC (2) - Say Yes.mp3

Sort by Title from id3tag means, that the second Audiobook from the Series would be placed before the first audiobook. 

That would be great if you could implace this Feature. 

Sorry for my Bad english.

**Comment by bjohas** [comment]
I would love to have this feature - also for folders. I make sure that all my folders are ordered meaningfully, but then it doesn't show up like that in Voice.
